Jetcaster/app/src/main/java/com/example/jetcaster/ui/player/PlayerViewModel.kt

```diff
@@ -99,4 +100,14 @@ class PlayerViewModel @Inject constructor(
     fun onRewindBy(duration: Duration) {
         episodePlayer.rewindBy(duration)
     }
+
+    fun onAddToQueue() {
+        uiState.episodePlayerState.currentEpisode?.let {
+            episodePlayer.addToQueue(it)
```

Comment:
are we adding every time the user taps the button? (just clarifying if you were going to add some logic on the episodePlayer for throttling or keep it as it is)

Reply:
Yep, we are adding the same episode to the queue as the user taps and there's no throttling/limiting behavior at the moment. If we want that we can add that logic in the `EpisodePlayer` but I would say that's out-of-scope for now.